### PR TITLE
Added lottie support + embeded 4 animation for intro-dialog

### PR DESCRIPTION
**Pull request by arielhasidim**

### CODE REVIEW

File: IsraelHiking.Web/src/application/application.module.ts [hunk 1]
```diff
@@ -37,6 +37,8 @@ import { NgxD3Service } from "@katze/ngx-d3";
 import { InfiniteScrollModule } from "ngx-infinite-scroll";
 import { NgxMapLibreGLModule } from "ngx-maplibre-gl";
 import { NgIdleModule } from "@ng-idle/core";
+import { LottieModule } from 'ngx-lottie';
+import player from 'lottie-web';
```
Comment (HarelM) [inline IsraelHiking.Web/src/application/application.module.ts:41]: Please run `npm run lint` to fix inconsistent quotes type.

File: IsraelHiking.Web/src/application/application.module.ts [hunk 2]
```diff
@@ -356,3 +359,7 @@ export class ApplicationModule {
         angulartics2GoogleAnalytics.startTracking();
     }
 }
+
+export function playerFactory() {
```
Comment (HarelM) [inline IsraelHiking.Web/src/application/application.module.ts:363]: I suggest to simply inline this: `() => player` in the above `forRoot(...`)`.

File: IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.html [hunk 3]
```diff
@@ -8,22 +8,30 @@ <h2>{{resources.language}}</h2>
                 </div>
             </mat-radio-group>
             <br />
-            <img src="content/default-poi.png" class="img-responsive" />
+            <div class="squareContainer">
+                <ng-lottie class="img-responsive" style="padding-right: 13px;" [options]="lottieLanguage"></ng-lottie>
```
Comment (HarelM) [inline IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.html:12]: `padding-right` will also work for English?

File: IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.ts [hunk 4]
```diff
@@ -6,12 +6,29 @@ import { NgRedux } from "../../reducers/infra/ng-redux.module";
 import { ResourcesService } from "../../services/resources.service";
 import { ApplicationState } from "../../models/models";
 import { ConfigurationActions } from "application/reducers/configuration.reducer";
+
+import { AnimationItem } from 'lottie-web';
```
Comment (HarelM) [inline IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.ts:10]: Same here for the quotes.

File: IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.ts [hunk 5]
```diff
@@ -6,12 +6,29 @@ import { NgRedux } from "../../reducers/infra/ng-redux.module";
...
+import { AnimationOptions } from 'ngx-lottie';
 @Component({
     selector: "intro-dialog",
-    templateUrl: "./intro-dialog.component.html"
+    templateUrl: "./intro-dialog.component.html",
+    styleUrls: ["./intro-dialog.component.scss"]
 })
 export class IntroDialogComponent extends BaseMapComponent {
 
+    lottieLanguage: AnimationOptions = {
```
Comment (HarelM) [inline IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.ts:19]: I'm not sure, but maybe you import the json file itself and use its content to solve the delay you noticed.
I'll take a look at it and let you know...

**Review by HarelM — CHANGES_REQUESTED** [review]
Thanks for this!!! Please add the relevant links to the attribution in this conversation so I add them to our attribution page.

File: IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.html (hunk 3 above)
Comment (arielhasidim) [reply IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.html:12]: yes

File: IsraelHiking.Web/src/application/application.module.ts (hunk 2 above)
Comment (arielhasidim) [reply IsraelHiking.Web/src/application/application.module.ts:363]: I'm leaving it to you (not sure how)

**Comment by arielhasidim** [comment]
> Thanks for this!!! Please add the relevant links to the attribution in this conversation so I add them to our attribution page.

**Attribution**
Scenery for POI:
[Animation by Sergey Riznyk on LottieFiles 
](https://lottiefiles.com/18981-warm-nature-scenery-with-trees) 
Language selection:
[Animation by Denis Duarte on LottieFiles 
](https://lottiefiles.com/40448-select-your-language)
Map:
[Animation by Isa Kadriu on LottieFiles ](https://lottiefiles.com/65400-map-location)
Planning:
[Animation by Guillaume Will on LottieFiles 
](https://lottiefiles.com/65931-plan)More (gift):
[Animation by Maximilien Goupil de Bouillé on LottieFiles ](https://lottiefiles.com/40253-gift-red)

and full [Creative Commons license](https://lottiefiles.com/page/license) page...

**Pull request merged**